### PR TITLE
Unable to create the instance of custom node right loaded from a file

### PR DESCRIPTION
**Pull request by marimano**
Open a .dyf file, select this node in the search list - an instance is added to Flood workspace, but it's absent in Dynamo.

It occurs because a corresponding search element has wrong data for creating the node and therefore Flood sends wrong data to Dynamo

### CODE REVIEW

File: app/scripts/models/App.js [hunk 1]
```diff
@@ -136,18 +136,20 @@ define(['backbone', 'Workspaces', 'Node', 'Login', 'Workspace', 'SearchElements'
 
     },
 
-    newNodeWorkspace: function( callback, customNodeName, silent ) {
+    newNodeWorkspace: function( callback, customNodeName, gotFromDynamo ) {
       this.context.createNewNodeWorkspace().done(function(data){
```
Comment (pboyer) [inline app/scripts/models/App.js]: Let's call this `lazyInit` rather than `gotFromDynamo`.

File: app/scripts/models/App.js [hunk 2]
```diff
@@ -136,18 +136,20 @@ define(['backbone', 'Workspaces', 'Node', 'Login', 'Workspace', 'SearchElements'
...
         data.isCustomNode = true;
         data.guid = this.makeId();
         data.name = customNodeName;
 
+        var attr = { app : this };
         // if we need to not send it to the dynamo
-        if (silent) {
+        if (gotFromDynamo) {
             data.notNotifyServer = true;
+            attr.initCustomNodeLater = true;
```
Comment (pboyer) [inline app/scripts/models/App.js]: Similarly, you could call this property `lazyInit` but it doesn't matter much.

**Comment by pboyer** [comment]
Thanks @marimano!  In Dynamo, custom nodes are lazily loaded.  Initially you just get basic information.  They aren't fully loaded/compiled until they're used.

**Comment by pboyer** [comment]
ping @marimano!

**Comment by marimano** [comment]
Hi @pboyer, I've resolved conflicts and made the small changes you asked

**Comment by pboyer** [comment]
Thanks @marimano!  Works perfectly!

**Pull request merged**